src/protocols/ec2-query.ts

```diff
@@ -1,21 +1,238 @@
+import { XMLParser } from "fast-xml-parser";
 import type { ProtocolHandler, ServiceMetadata } from "./interface.ts";
-import { parseEC2Response } from "../ec2-parsers.js";
+
+let ec2ModelMeta: any = null;
+
+// Try to load the metadata synchronously at module load time
+try {
+  // Use dynamic import but don't await it - this will fail gracefully
+  import("../ec2-metadata.js")
+    .then(({ ec2ModelMeta: meta }) => {
+      ec2ModelMeta = meta;
+    })
+    .catch(() => {
+      // Metadata not available, will use fallback
+      ec2ModelMeta = null;
+    });
+} catch {
+  // Module not found or error during import
+}
```

Comment:
Why do we want to swallow this error?

Reply:
yeah, we don't want to swallow that. it previously had some junk as "fallback", and fallback isn't possible. I needed to research more about dynamic imports and how they worked ... anyway, updated it now to throw a hard error if it can't load the metadata.